cli/crates/cli/src/output/report.rs

```diff
@@ -377,6 +376,22 @@ pub fn delete_branch_success() {
     watercolor::output!("\n✨ The branch was successfully deleted!", @BrightBlue);
 }
 
+pub fn delete_env_var() {
+    watercolor::output!("🕒 Environment variable is being deleted...", @BrightBlue);
+}
+
+pub fn delete_env_var_success() {
+    watercolor::output!("\n✨ The environment variable was successfully deleted!", @BrightBlue);
+}
+
+pub fn create_env_var() {
+    watercolor::output!("🕒 Environment variable is being created...", @BrightBlue);
```

Comment:
I wonder if we can omit the created... message since we show success/fail after  anyway?